### PR TITLE
Fix wrong operator bug in PyCMSError exception message

### PR DESCRIPTION
**Pull request by LostBenjamin**
Hi,

This pull request is a fix to a potential wrong operator bug at `src/PIL/ImageCms.py`. Please check the changes.

Best,
Jingxuan

### CODE REVIEW

**Comment by hugovk** [comment]
Thanks for the PR. Please could you add a test for this? And let's also use an f-string.

**Comment by LostBenjamin** [comment]
You are welcome.

This bugs was found by our automated tool statically. I am not familiar with the codebase so it would be difficult for me to write a test case.

**Comment by hugovk** [comment]
That's fine, I've added tests, thanks!

File: Tests/test_imagecms.py [hunk 1]
```diff
@@ -174,19 +174,24 @@ def test_exceptions():
...
         ImageCms.buildTransform("foo", "bar", "RGB", "RGB")
-    with pytest.raises(ImageCms.PyCMSError):
+
+    with pytest.raises(ImageCms.PyCMSError, match="Invalid type for Profile"):
         ImageCms.getProfileName(None)
     skip_missing()
-    with pytest.raises(ImageCms.PyCMSError):
+
+    # macOS/Ubuntu: "'NoneType' object cannot be interpreted as an integer"
+    # Windows: "an integer is required (got type NoneType)"
```
Comment (radarhere) [inline Tests/test_imagecms.py:193]: This doesn't seem [correct.](https://github.com/radarhere/Pillow/runs/6953537293?check_suite_focus=true#step:24:1570)

Comment (hugovk) [reply Tests/test_imagecms.py:193]: Ah, interesting, different messages on Windows depending on version.

3.7-3.9:

```
E           PIL.ImageCms.PyCMSError: an integer is required (got type NoneType)
```

https://github.com/radarhere/Pillow/runs/6953537155?check_suite_focus=true#step:24:1570
https://github.com/radarhere/Pillow/runs/6953537211?check_suite_focus=true#step:24:1570
https://github.com/radarhere/Pillow/runs/6953537251?check_suite_focus=true#step:24:1570

3.10:

```
E           PIL.ImageCms.PyCMSError: 'NoneType' object cannot be interpreted as an integer
```

https://github.com/radarhere/Pillow/runs/6953537293?check_suite_focus=true#step:24:1570

File: Tests/test_imagecms.py [hunk 2]
```diff
@@ -174,19 +174,24 @@ def test_exceptions():
...
         ImageCms.buildTransform("foo", "bar", "RGB", "RGB")
-    with pytest.raises(ImageCms.PyCMSError):
+
+    with pytest.raises(ImageCms.PyCMSError, match="Invalid type for Profile"):
         ImageCms.getProfileName(None)
     skip_missing()
-    with pytest.raises(ImageCms.PyCMSError):
+
+    # Windows, Python <= 3.9: "an integer is required (got type NoneType)"
+    # Others: "'NoneType' object cannot be interpreted as an integer"
```
Comment (radarhere) [inline Tests/test_imagecms.py:193]:
```suggestion
    # Python <= 3.9: "an integer is required (got type NoneType)"
    # Python > 3.9: "'NoneType' object cannot be interpreted as an integer"
```
See macOS [3.8](https://github.com/radarhere/Pillow/runs/6953537130?check_suite_focus=true#step:8:1590), [3.9](https://github.com/radarhere/Pillow/runs/6953537092?check_suite_focus=true#step:8:1590) and [3.10](https://github.com/radarhere/Pillow/runs/6953537060?check_suite_focus=true#step:8:1589), Ubuntu [3.8](https://github.com/radarhere/Pillow/runs/6953537303?check_suite_focus=true#step:8:1591), [3.9](https://github.com/radarhere/Pillow/runs/6953537279?check_suite_focus=true#step:8:1591) and [3.10](https://github.com/radarhere/Pillow/runs/6953537256?check_suite_focus=true#step:8:1590)

**Review by radarhere — APPROVED** [review]

**Pull request merged**